sdk/core/azure-core/src/main/java/com/azure/core/http/rest/PagedFlux.java

```diff
@@ -58,6 +58,16 @@ public PagedFlux(Supplier<Mono<PagedResponse<T>>> firstPageRetriever,
         this.nextPageRetriever = nextPageRetriever;
     }
 
+    /**
+     * Creates an instance of {@link PagedFlux} using argument as a supplier that fetches the first page of {@code T}.
+
+     * <p><strong>Code sample</strong></p>
```

Comment:
Is there a code snippet or a code sample that's missing here?

Reply:
This constructor is removed.